src/loaders/TextureLoader.js

```diff
@@ -21,7 +22,8 @@ TextureLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 		var texture = new Texture();
 
-		var loader = new ImageLoader( this.manager );
+		var isWorker = typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope;
+		var loader = isWorker ? new ImageBitmapLoader( this.manager ) : new ImageLoader( this.manager );
```

Comment:
I'm afraid this change is incorrect since `ImageBitmapLoader` behaves differently than `ImageLoader`. They are not interchangeable.

Reply:
According to https://threejs.org/docs/#api/en/loaders/ImageBitmapLoader It should work that same, but sure ImageBitmapLoader has some issues, but they are better than broken TextureLoader inside WebWorker. 

Do you have any ideas about how It can be fixed?

Reply:
I dug deeper and find that it isn't safe to modify TextureLoader.

I moved my changes outside of TextureLoader to CanvasTextureLoader.